docs/img/omnisci/omnisci_lazy_tree_example.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Do not edit this file with editors other than diagrams.net -->
```

Comment:
pls try to make it narrower but taller, e.g. by placing frame nodes and expression nodes boxes under each other or by placing inner FrameNodes and InputRefExpr boxes under each other instead of side by side

Reply:
[done](https://modin--3753.org.readthedocs.build/en/3753/flow/modin/experimental/core/execution/native/implementations/omnisci_on_native/index.html#lazy-execution)